esp/esp/survey/views.py

```diff
@@ -37,6 +37,7 @@
 
 import datetime
 import xlwt
+import re
```

Comment:
I don't understand what this is, and the use of anchors makes it clear that no one has used it in years. Objections to deleting?